ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```diff
@@ -878,6 +878,14 @@ $defs:
               This setting is only applied on the devices where it is set, it does not automatically affect any parent/child devices configuration, so it must be set on each applicable node/node-group/node-type as needed.
             type: int
             default: 1500
+          inband_ztp_parent:
```

Comment:
wouldn't it be simpler if we set this on the device to be booted via inband_ztp? Then we can check for inband_mgmt_vlan and raise if it is not set or if the device type is wrong.
It would also be simpler to implement inband ZTP via MLAG peer link when it is a setting on the device instead of the parent.